docs/content/documentation/content/overview.md

```diff
@@ -63,3 +63,14 @@ the public web site. You can achieve this by simply setting `ignored_content` in
 ```
 ignored_content = ["*.xlsx"]
 ```
+
+## Static assets
+
+In addition to placing content files in the `content` directory, you may also place content 
+files in the `static` directory.  Any files/folders that you place in the `static` directory
+will be copied, without modification, to the public directory.  
+
+Typically, you might put site-wide assets (such as the site favicon, site logos or site-wide
+JavaScript) in the root of the static directory.  You can also place any HTML or other files that
+you wish to be included without modification (that is, without being parsed as Markdown files)
+into the static directory.
```

Comment:
The last sentence is not 100% true, you can also put non-md files in `content` as colocated-assets (https://www.getgutenberg.io/documentation/content/overview/#assets-colocation) as mentioned above in that file and they won't be modified either.

Reply:
I added an example that clarifies that you can either colocate files or add them to the static folder, depending on which system fits your desired organazational structure.  Please let me know if you would like me to revise further.